codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/presignable/PresignableUrlIntegration.kt

```diff
@@ -144,6 +150,40 @@ class PresignableUrlIntegration(private val presignedOperations: Map<String, Set
         }
     }
 
+    private fun renderPresignURLAPIInServiceClient(
+        writer: SwiftWriter,
+        clientName: String,
+        op: OperationShape,
+        inputType: String
+    ) {
+        writer.apply {
+            openBlock("extension $clientName {", "}") {
+                val params = listOf("input: $inputType", "expiration: Foundation.TimeInterval")
+                val returnType = "Foundation.URL"
+                renderDocForPresignURLAPI(this, op, inputType)
+                openBlock("public func presignURLFor${op.toUpperCamelCase()}(${params.joinToString()}) async throws -> $returnType {", "}") {
```

Comment:
Function name should start with `presignedURLFor...`, not `presignURLFor`.  Same for the request functions.

Reply:
Changed as suggested.

Reply:
Same here, rendered function is still named `presignURLFor...`